Automattic/jetpack
# Settings menu: add Jetpack item for sites with Scan product

Fixes 1164141197617539-as-1200624737982594

#### Changes proposed in this Pull Request:
This PR adds the _Jetpack_ item in the settings menu for sites that have subscribed to a Scan product. The item used to be visible with plans including Scan, but not with a Scan subscription only.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

**Prerequisites**

- Download the PR
- Launch your local testing site and make sure it has the local version of the plugin installed
- Make sure your testing site has no Jetpack subscription

**Testing Calypso**
- Visit `wordpress.com` and select your testing site
- Check that the _Settings_ menu doesn't have the _Jetpack_ item
- Purchase a _Scan_ subscription
- Check that now the _Settings_ menu do show the _Jetpack_ item

<img width="447" alt="Screen Shot 2021-11-29 at 1 51 51 PM" src="https://user-images.githubusercontent.com/1620183/143925942-654f662a-459c-4a6d-ae1e-77db90d21800.png">
